typescript/core/src/ChunkCursor.ts

```diff
@@ -26,11 +27,9 @@ export class ChunkCursor {
   #reverse: boolean;
 
   // List of message offsets (across all channels) sorted by logTime.
-  #orderedMessageOffset?: [logTime: bigint, offset: bigint][];
+  orderedMessageOffsets?: [logTime: bigint, offset: bigint][];
```

Comment:
Did you mean to make public by omitting the `#`?

Reply:
ah, no! good catch